spec/phone_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'phone'
+require 'takeaway'
+
+describe Phone do
+  describe '#receive' do
+    it 'should receive a text message' do
+      text_double = double :text, text: "Thank you! Your order was placed and will be delivered before 18:00"
```

Comment:
Since you'll move the Twilio code into `Phone`, you'll also have to modify this test so the class from the Twilio gem is mocked — and the job of this test will be to **check that the `Phone` class correctly called the Twilio library** — you can use this guidance about how to stub external libraries: https://github.com/makersacademy/course/blob/main/pills/levels_of_stubbing.md#2-stub-the-gem-associated-with-3rd-party-api